src/protocolsupport/protocol/pipeline/version/v_pe/ToClientPacketEncoder.java

```diff
@@ -48,10 +55,40 @@
 		registry.register(TabCompleteResponse.class, NoopWriteablePacket.class);
 		registry.register(BossBar.class, NoopWriteablePacket.class);
 		registry.register(Title.class, NoopWriteablePacket.class);
+		registry.register(PluginMessage.class, CustomEventPacket.class);
 	}
 
 	public ToClientPacketEncoder(Connection connection, NetworkDataCache cache) {
 		super(connection, cache);
 	}
 
+	// wish java had some kind of real 'tuple' class
```

Comment:
ProtocolSupport has a touble class wich you can copy over to psbungee if you need.

Reply:
hmmm not sure im seeing it in PSbungee?

Reply:
"you can copy it over to PSB"

Reply:
looks like PS uses the apache commands tuple, which apparently extends Map.Entry anyways lol. ill switch over to it tho, looks cleaner